todo_app/data/classes.py

```diff
@@ -0,0 +1,6 @@
+class to_do_item:
+    def __init__(self,id, title , status):
+        self.id = id
+        self.title = title
+        self.status = status
```

Comment:
Minor, it's nicer if we have a `from_trello_card` function that would make it easier to change from a trello card to this class